src/pages/why-choose-us/index.tsx

```diff
@@ -89,6 +92,7 @@ const WhyChooseUs = () => {
             <Section>
                 <IconGrid />
             </Section>
+            {is_latam ? <LatamAward /> : <></>}
```

Comment:
How about we render it like so? 
```suggestion
            {is_latam && <LatamAward />}
```